cmd/cdi-importer/BUILD.bazel

```diff
@@ -4,40 +4,35 @@ load("@io_bazel_rules_container_rpm//rpm:rpm.bzl", "rpm_image")
 rpm_image(
     name = "cdi-importer-base-image",
     base = select({
-        "@io_bazel_rules_go//go/platform:linux_arm64": "@fedora-aarch64//image",
-        "//conditions:default": "@fedora//image",
+        "@io_bazel_rules_go//go/platform:linux_arm64": "@centos-aarch64//image",
+        "//conditions:default": "@centos//image",
     }),
     rpms = select({
         "@io_bazel_rules_go//go/platform:linux_arm64": [
             "@liburing-aarch64//file",
             "@libnbd-aarch64//file",
-            "@libxcrypt-compat-aarch64//file",
             "@nbdkit-server-aarch64//file",
             "@nbdkit-basic-filters-aarch64//file",
+            #"@nbdkit-vddk-plugin-aarch64//file",
```

Comment:
Don't we need the vddk plugin?

Reply:
The plugin doesn't seem to be built for aarch64 - I'm not sure why that is the case.
(I didn't notice I'm the one introducing this change, I thought it's an existing one)

Reply:
Worth noting that vmware aarch64 support is in tech preview and not really likely to be in heavy use